src/common.ts

```diff
@@ -44,6 +44,20 @@ export function isR<T = DataSource>(obj: unknown): obj is ReactiveProxy<T> {
   )
 }
 
+/**
+ * Utility that ensures we only attempt to make reactive objects that _can_ be made reactive.
+ *
+ * Examples of objects that cause issues: NodeList, HTMLElement
+ * @see {@link https://github.com/vuejs/core/blob/8998afa42755cbdb3403cd6c0fe158980da8492c/packages/reactivity/src/reactive.ts#L43-L62}
+ */
+export function canReactiveWrap(maybeObj: any): boolean {
+  return ['Object', 'Array', 'Map', 'Set', 'WeakMap', 'WeakSet'].includes(
```

Comment:
it looks like Map, Set, WeakMap and WeakSet don't work when wrapped so might be worth removing them from here
```suggestion
  return ['Object', 'Array'].includes(
```